### ISSUE TITLE
Players failing to authenticate with steam

### ISSUE THREAD

**Issue by Wheretf2** [sapphonie/StAC-tf2#191]
Hello,
﻿
I'm running a [retry on restart plugin](https://forums.alliedmods.net/showthread.php?t=202625) that seems to be conflicting with StAC
﻿
When the server is restarted, players that automatically connect are not verified through steam and have their steamID reported as "STEAM_ID_STOP_IGNORING_RETVALS" through client console, which the server also seems to be working off of; ex for bans it will report the steamID as the above which naturally is an issue
﻿
It's worth noting the server console status command will report the correct uniqueid, but the client console will not

**Comment by sapphonie** [comment]
Sydney
 — 
04/05/2024 6:58 PM
code for that plug-in looks innocent, though I didn’t know about _restart as I use quit; does that behave differently?


sappho
 — 
Yesterday at 4:12 PM
i dont know if this is a stac issue frankly
stop ignoring retvals is a sourcemod thing
i dont know how that would end up in client console

**Comment by sapphonie** [comment]
@Wheretf2 Run `version; plugin_print; meta version; meta list; sm version; sm exts list; sm plugins list` in server console and post output here please. But first, let me know if this is still an issue with 6.1.4. Ty

**Comment by Wheretf2** [comment]
Tested with 6.1.4, issue persists

Build Label:           8622567   # Uniquely identifies each build
Network PatchVersion:  8622567   # Determines client and server compatibility
Protocol version:           24   # High level network protocol version
Server version:        8622567
Server AppID:           232250

Loaded plugins:
---------------------
0:      "Metamod:Source 2.0.0-dev+1284"
---------------------

 Metamod:Source Version Information
    Metamod:Source version 2.0.0-dev+1284
    Plugin interface version: 16:14
    SourceHook version: 5:5
    Loaded As: Valve Server Plugin
    Compiled on: Mar  6 2024 19:17:22
    Built from: https://github.com/alliedmodders/metamod-source/commit/9fbe7f3
    Build ID: 1284:9fbe7f3
    http://www.metamodsource.net/

 SourceMod Version Information:
    SourceMod Version: 1.12.0.7114
    SourcePawn Engine: 1.12.0.7114, jit-x86 (build 1.12.0.7114)
    SourcePawn API: v1 = 5, v2 = 16
    Compiled on: Mar 20 2024 17:14:43
    Built from: https://github.com/alliedmodders/sourcemod/commit/dbec0b16
    Build ID: 7114:dbec0b16
    http://www.sourcemod.net/

Waiting on approval to post other info if needed

Thank you!

**Comment by sapphonie** [comment]
Unload `SMRCon` (which is no longer needed or useful as far as I know) and sigmod (which I'm pretty sure has a similar hook to Connect), and see if this still persists. Also, SourceMod 1.12 isn't stable, and could definitely be causing problems as well.

**Comment by Wheretf2** [comment]
Hi,

Sorry for the delay, but wanted to update this issue that the conflict was related to "[09] Connect (1.2.1) by Asher "asherkin" Baker"
Updating to 1.2.0 resolved the issue

Thank you!

**Comment by sapphonie** [comment]
@Wheretf2 you mean 1.4.0? this issue should be fixed with stac 6.1.5, which updates connect as well
